src/nvim/edit.c

```diff
@@ -4096,7 +4111,14 @@ int ins_compl_add_tv(typval_T *const tv, const Direction dir, bool fast)
     return FAIL;
   }
   return ins_compl_add((char_u *)word, -1, NULL,
-                       (char_u **)cptext, true, &user_data, dir, flags, dup);
+                       (char_u * *)cptext, true, &user_data, dir, flags, dup);
+}
+
+/// Returns TRUE when using a user-defined function for thesaurus completion.
+static int thesaurus_func_complete(int type)
```

Comment:
```
/// Returns true when using a user-defined function for thesaurus completion.
static bool thesaurus_func_complete(int type)
```

Reply:
`type` should be an enum/typedef.

Reply:
@janlazo Changing `int` to `bool` is simple, but should I create that `enum` or will we wait on the upstream?

Reply:
Create it yourself. You can do this in a future PR.

Reply:
@seandewar BTW, I am not the author of the original vim patch. But I guess it means, that this conversation can be marked as resolved, right?